web/blob/static/js/status.js

```diff
@@ -0,0 +1,26 @@
+/* globals PATH_PREFIX, jQuery */
+(function ($, undefined) {
+    'use strict';
+
+    $('#reload-conf-btn').on('click', function () {
```

Comment:
For better or worse, we've been using underscores for classes/IDs that we define ourselves (vs. ones from e.g. Bootstrap). Could make change to update that all at once to dashes at some point, but for now it'd be better to keep it consistent with our other names and use underscores.